.editorconfig

```diff
@@ -10,6 +10,6 @@ charset = utf-8
 indent_style = space
 indent_size = 2
 
-[{package.json,*.yml,*.cjson}]
+[{*.json,*.yml,*.cjson}]
```

Comment:
YAML has 2 file extensions, `*.yml` and `*.yaml`.
So, it would be better to handle both file extensions in all files

- `.editorconfig`
- `.eslintignore`